aten/src/ATen/common_with_cwrap.py

```diff
@@ -190,6 +190,13 @@ def parse_header(path):
             else:
                 fn_name = fn_name[:-1]
             generic_functions.append(Function(fn_name))
+        elif l.startswith('THC_API void THNN_'):
+            fn_name = l.lstrip('THC_API void THNN_')
+            if fn_name[0] == '(' and fn_name[-2] == ')':
+                fn_name = fn_name[1:-2]
+            else:
+                fn_name = fn_name[:-1]
+            generic_functions.append(Function(fn_name))
```

Comment:
Why do we need this now?

Reply:
I dunno if we need it or not, but I was fixing places where we hardcoded `TH_API` (because I flipped a bunch of these to be `THC_API`)